frontend/public/components/namespace.jsx

```diff
@@ -92,7 +92,7 @@ const ProjectRow = ({obj: project}) => {
     <div className="col-md-3 col-sm-3 col-xs-4">
       <StatusIcon status={project.status.phase} />
     </div>
-    <div className="col-md-3 col-sm-3 hidden-xs">
+    <div className="col-md-3 col-sm-3 hidden-xs co-break-word">
```

Comment:
There's a fix for this in #865 (although break word might be better than truncate)